LoopKitUI/Views/FractionalQuantityPicker.swift

```diff
@@ -216,7 +216,7 @@ fileprivate extension Collection where Element == Decimal {
     /// - Precondition: The collection is sorted in ascending order.
     func deltaScale(boundedBy maxScale: Int) -> Int {
         let roundedToMaxScale = lazy.map { $0.rounded(toPlaces: maxScale) }
-        guard let maxDelta = roundedToMaxScale.adjacentPairs().map(-).map(abs).max() else {
+        guard let maxDelta = roundedToMaxScale.adjacentPairs().map(-).map(abs).min() else {
```

Comment:
I was surprised that this was wrong, but I tested it, and I think you're right.  It should be using the smallest delta to determine the number of decimal places.

The name for the var should probably be changed to `minDelta`.  And there appears to be a copy of this in the watch app that should also be updated.

Reply:
oh... right :)

Reply:
@ps2 just noticed your note about the watch app copy. Any hints as to where I can find this or what copy I'm looking for?

Reply:
oh I think I found it.